pkgs/by-name/gy/gysmo/package.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, stdenv, fetchFromGitHub, go }:
+
+stdenv.mkDerivation rec {
+  pname = "gysmo";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "grosheth";
+    repo = "gysmo";
+    rev = "v${version}";
+    sha256 = "ucDcEygh315yD999z6W+kqNBZLj0lGuxZUYjSkOhcJw=";
+  };
+
+  buildInputs = [ go ];
+
+  buildPhase = ''
```

Comment:
please add `runHook preBuild`, `runHook postBuild`, `runHook preInstall`, `runHook postInstall` to facilitate user override